src/context-selection/org-unit-selector-bar-item/use-organisation-unit-levels.js

```diff
@@ -0,0 +1,16 @@
+import { useQuery } from 'react-query'
+
+export default function useOrganisationUnitLevels() {
```

Comment:
I think we should reorganize the query structuring a little for this PR. You could simplify things a lot with dependent queries and a query key factory. Probably best if we plan a short google meet for that, easier to explain it that way.

Reply:
Refactored as discuss